tests/unit/src/main/scala/tests/BaseDigestSuite.scala

```diff
@@ -51,7 +52,7 @@ trait BaseDigestSuite extends BaseSuite {
           if (isEqual) {
             assertNoDiff(x, y)
           } else {
-            assertNotEquals(x, y, "The digests should not be equal")
+            assert(x != y, "The digests should not be equal")
```

Comment:
I think this should be assertDiffNotEqual (it was probably a leftover from when I initially decided to remove that assertion)